llvm/lib/IR/OptBisect.cpp

```diff
@@ -32,6 +32,11 @@ static cl::opt<int> OptBisectLimit("opt-bisect-limit", cl::Hidden,
                                    }),
                                    cl::desc("Maximum optimization to perform"));
 
+static cl::opt<bool> OptBisectPrintPassMessage(
+    "opt-bisect-print-pass-message",
```

Comment:
verbose?

Reply:
I didn't follow

Reply:
print-pass-message is an awkward flag name. usually would be a verbose flag